README.md

```diff
@@ -0,0 +1,880 @@
+# Testaframe - Test Automation Framework
+
+
+## Overview
+
+Testaframe can be used for driving Selenium tests and/or API/service
+tests. Two of its main features are the ability to run tests on multiple
+OS/browsers across multiple environments (e.g. localhost, CI, QA) at
+once, and polling asserts, which is especially useful for testing Ajaxy
+web sites and asynchronous services.
+
+There is still a bit of work and documentation to do before this is all
+pretty and easily used and learned by new people, but it is better to
+get it committed and public rather than wait until it is perfect.
+
+This README can be converted to html using the `Makefile`
+
+``` bash
+sudo pip install docutils  # if not already installed
+make docs
+```
+
+## Getting Started via Docker
+
+### Requirements
+
+Install Docker for your platform:
+- [Docker for Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+
+- [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
+
+- [Docker for Windows](https://docs.docker.com/docker-for-windows/install/)
+
+Clone the [testaframe repo](https://github.com/emailit2me2/testaframe)
+to your local box.
+
+### Setting up Testaframe
+
+First, make a copy of `run_local.py` as `run_user.py`.
+
+``` bash
+cp run_local.py run_user.py
+```
+
+Next, you will need to create a personal `my_cfg.py` on your local machine by
+using `my_cfg_example.py` as an example. Create one using
+
+``` bash
+cp my_cfg_example.py my_cfg.py
+```
+
+The `my_cfg.py` file should be .ignored in your source control.
+
+Setting up the Docker environment is simple.
+
+``` bash
+docker-compose up
+```
+
+To run the tests:
+
+``` bash
+docker-compose exec app ./run_tests.sh -s -v -a Wiki
```

Comment:
Running this out of the box yielded an error.
  File "/home/seluser/testaframe/config/env.py", line 167, in get_se_driver
    opts = Options()
NameError: global name 'Options' is not defined

Reply:
Oh fun I must have a pip package installed or something . I'll figure it out as it worked on my box :/